typescript/infra/src/core/deploy.ts

```diff
@@ -202,15 +226,33 @@ export class AbacusCoreDeployer extends AbacusAppDeployer<
   ): Promise<ethers.ContractReceipt> {
     const contracts = core.mustGetContracts(domain);
     const overrides = core.getOverrides(domain);
-    await contracts.validatorManager.transferOwnership(owner, overrides);
+    await contracts.outboxMultisigValidatorManager.transferOwnership(
+      owner,
+      overrides,
+    );
     await contracts.xAppConnectionManager.transferOwnership(owner, overrides);
     await contracts.upgradeBeaconController.transferOwnership(owner, overrides);
     for (const chain of Object.keys(
       contracts.addresses.inboxes,
     ) as ChainName[]) {
+      await contracts
+        .inboxMultisigValidatorManager(chain)
+        .transferOwnership(owner, overrides);
       await contracts.inbox(chain).transferOwnership(owner, overrides);
     }
     const tx = await contracts.outbox.transferOwnership(owner, overrides);
     return tx.wait(core.getConfirmations(domain));
   }
+
+  multisigValidatorManagerConfig(
+    config: CoreConfig,
+    domain: types.Domain,
+  ): MultisigValidatorManagerConfig {
+    const domainName = this.mustResolveDomainName(domain);
+    const validatorManagerConfig = config.multisigValidatorManagers[domainName];
+    if (!validatorManagerConfig) {
+      throw new Error(`No validator manager config for ${domainName}`);
+    }
```

Comment:
isn't `CoreConfig` typed to prevent this? or are we passing that config from outside of typescript